app/src/main/java/org/mozilla/fenix/home/collections/CollectionMenu.kt

```diff
@@ -0,0 +1,142 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.home.collections
+
+import android.content.Context
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.DropdownMenu
+import androidx.compose.material.DropdownMenuItem
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.DisposableEffect
+import androidx.compose.runtime.Immutable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import mozilla.components.browser.state.selector.normalTabs
+import mozilla.components.feature.tab.collections.TabCollection
+import org.mozilla.fenix.R.string
+import org.mozilla.fenix.ext.components
+import org.mozilla.fenix.home.sessioncontrol.CollectionInteractor
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Menu shown for a [TabCollection].
+ *
+ * @see [DropdownMenu]
+ *
+ * @param showMenu Whether this is currently open and visible to the user.
+ * @param menuItems List of options shown.
+ * @param onDismissRequest Called when the user chooses a menu option or requests to dismiss the menu.
+ */
+@Composable
+fun CollectionMenu(
```

Comment:
can we get a `@Preview` for this?

Reply:
I've had this issue occur usually because of the `components` object, in case that's what's causing it here. I may poke around later to also take a look.